tldw_Server_API/app/core/DB_Management/Jobs_Repository.py

```
@@ -0,0 +1,240 @@
+        *,
+        backend: str,
+        db_path: Path | None = None,
+        db_url: str | None = None,
+    ) -> None:
+        """Build a repository bound to either a SQLite path or Postgres DSN."""
+        self.backend = backend
+        self.db_path = Path(db_path) if db_path is not None else None
+        self.db_url = db_url
+
+    @classmethod
+    def for_sqlite(cls, db_path: Path) -> "JobsRepository":
```

Ruff (0.15.6)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)